src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -15,8 +15,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="CompressedStackTests.cs" />
     <Compile Include="ExceptionTests.cs" />
     <Compile Include="ThreadExceptionEventArgsTests.cs" />
+    <Compile Include="ThreadTests.netstandard1.7.cs" />
```

Comment:
I thought about that, but since the Thread type existed previously, we may in the future add tests for preexisting functionality in ThreadTests.cs and tests for new APIs in the .netstandard1.7 file.